Source/DafnyLanguageServer.Test/DafnyLanguageServer.Test.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.7.0" />
```

Comment:
Where is this dependency used?

Reply:
It's used in the method `CreateDotNetDllThatStartsGivenFilepath` where it compiles the inline C# code.